cmake/Hyperscan.cmake

```diff
@@ -1,9 +1,6 @@
 option (ENABLE_HYPERSCAN    "Enable hyperscan for fast regexp processing [default: OFF]" OFF)
 
 if (ENABLE_HYPERSCAN MATCHES "ON")
-    if (NOT ("${ARCH}" STREQUAL "x86_64" OR "${ARCH}" STREQUAL "i386"))
-        MESSAGE(FATAL_ERROR "Hyperscan is supported only on x86_64/i386 architectures")
-    endif ()
```

Comment:
Shouldn't we limit it to x86_64, i386 and aarch64 only? I'm not quite sure about that, so it might be removed indeed...

Reply:
I was going to do that, but for compiling locally my SBC it appears as arch `unknown` in cmake instead of `aarch64`.  That is inside an alpine docker container.